### DISCUSSION TITLE
Need help with installlation

### DISCUSSION THREAD

**Discussion by fishhhead** [OpenTalker/SadTalker#616] [General]
![image](https://github.com/OpenTalker/SadTalker/assets/41725132/6589fb89-2135-4be7-8a5b-cd37e8d02d67)

Can anyone help me with this please? Thanks

**Comment by 3Diva** [comment]
I wish I could help. I'm having trouble myself, except I installed it as a WebUI AUTOMATIC1111 extension and it just gives me error messages when I try to generate a video with it. I hope you and I both get the issues figured out! It would be great to be able to use SadTalker, as it seems like it would be a helpful tool to have.
